website/client/components/chat/chatCard.vue

```diff
@@ -248,6 +248,16 @@ export default {
       return achievementsLib.getContribText(message.contributor, message.backer) || '';
     },
   },
+  mounted () {
+    const links = this.$refs.markdownContainer.getElementsByTagName('a');
+    for (let i = 0; i < links.length; i++) {
+      const link = links[i];
+      links[i].onclick = () => {
+        event.preventDefault();
+        this.$router.push({ path: link.getAttribute('href')});
```

Comment:
just curious, is this necessary? i mean keeping a normal `a` tag with no `onclick` handler doesn't work?

Reply:
makes sense, thanks!

Reply:
Actually I just tested this and it still opens the new tab, you are missing the event in the function definition: `links[i].onclick = () => {` ->  `links[i].onclick = (event) => {`

Reply:
(and it should not be called `event` but `ev` or something else because `event` is defined in the global scope so it will throw an eslint error)